r2/r2/lib/menus.py

```diff
@@ -609,7 +605,7 @@ def make_title(self, attr):
 class SearchSortMenu(SortMenu):
     """Sort menu for search pages."""
     _default = 'relevance'
-    mapping = search_sorts
+    mapping = g.search.sorts
```

Comment:
Note: I realize that it *appears* that `cls.mapping` is used below; but from what I can tell, that is dead code; there's no `cls.default`\* defined so that code would crash.

\* `cls._default`, with an underscore *is* defined and used elsewhere.